### ISSUE TITLE
Advice

### ISSUE THREAD

**Issue by stefandevo** [jonataslaw/get#134]
Hi
Keep up the good work! This lib is really interesting while some others in the community try to "talk it down" like in the Reddit conversation... 
But a small thing; try to use things like github actions to streamline deployment. Often we see old code here, newer versions op pub.dev, older versions on the release tab etc. That gives not a good feeling when looking to the repo. Current version tab says 2.0, pub.dev sas 2.5.9; code is now also up to date to 2.5.9.
Also include a link to the pub.dev page here in the readme (or in the description).

**Comment by jonataslaw** [comment]
This week this library went up 100 stars, and with the popularity there has also been a wave of haters, especially people who are creators of some package that has some of the functions of Get.
I understand your concern, and I give you reason, but creating something that has not yet been created (a complete solution that includes route, instance and state management) has its price. I have dedicated myself as much as I can to keep the library free of bugs, with maximum performance, but obviously I can't cover everything, the documentation doesn't have all the resources, sometimes I was logged into my web development github (I I also have two accounts because the other's audience is only for web development), and I needed to send a hotfix to the pub, and with that the master version was late, but that doesn't happen anymore.
I usually just launch the main versions in the "release" tab, which involve breaks or major resources. I'm afraid of polluting my followers' timeline with 2, sometimes even 4 updates on the same day. Maybe if this is so necessary, I can review it and add the releases on github with each update.
Obviously, even if I do all of this, some people will continue to fight against the package, and others will continue to know and use it, but constructive criticism is always welcome.
